website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx

```diff
@@ -76,30 +76,38 @@ This POST endpoint requires a JSON object with the following properties as a req
+| Key path                                                            | Type            | Default | Description                                                                                                                                                      |
+| ------------------------------------------------------------------- | --------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                                                         | string          |         | Must be `"tasks"`.                                                                                                                                               |
+| `data.attributes.name`                                              | string          |         | The name of the task. Can include letters, numbers, `-`, and `_`.                                                                                                |
+| `data.attributes.url`                                               | string          |         | URL to send a run task payload.                                                                                                                                  |
+| `data.attributes.description`                                       | string          |         | The description of the run task. Can be up to 300 characters long including spaces, letters, numbers, and special characters.                                    |
+| `data.attributes.category`                                          | string          |         | Must be `"task"`.                                                                                                                                                |
+| `data.attributes.hmac-key`                                          | string          |         | (Optional) HMAC key to verify run task.                                                                                                                          |
+| `data.attributes.enabled`                                           | bool            | true    | (Optional) Whether the task will be run.                                                                                                                         |
+| `data.attributes.global-configuration.enabled` **(beta)**           | bool            | false   | (Optional) Whether the task will be associated on all workspaces.                                                                                                |
+| `data.attributes.global-configuration.stages` **(beta)**            | array of string |         | (Optional) The stages of the run lifecycle when the run task should begin. Must be one or more of `"pre_plan"`, `"post_plan"`, `"pre_apply"`, or `"post_apply"`. |
```

Comment:
```suggestion
| `data.attributes.global-configuration.stages` **(beta)**            | array[string] |         | (Optional) The stages of the run lifecycle when the run task should begin. Must be one or more of `"pre_plan"`, `"post_plan"`, `"pre_apply"`, or `"post_apply"`. |
```

Reply:
So,  the change to `array[string]` did get me thinking. That's more a programming style instead of a more plain english style. I went searching through other documents looking for prior art and I found that both my original change and your suggestion are unique. Instead I found this usage a couple of times;

`| .... | array | An array of <blah> which .... |`

See `data.attributes.triggers` https://developer.hashicorp.com/terraform/cloud-docs/api-docs/notification-configurations#request-body

Would this be better to maintain consistency?

Reply:
You raise a very good point! I had pulled the `array[string]` style from another document, but we're actually in the process of formalizing these sort of style decisions. I spoke with a couple other team members who pointed me in the direction of some material for how we plan on standardizing these types across our documentation.

In the mean time, I agree with your point, we'll update my suggestions. Thanks!